### PR TITLE
Use pip to install numpy and scipy

### LINKED ISSUE #5: Upgrade to Numpy 1.9.2
https://github.com/numpy/numpy/tree/v1.9.2

### PR DESCRIPTION
**Pull request by cancan101**
Closes #5.

### CODE REVIEW

File: scripts/install-deeplearning-libraries.sh [hunk 1]
```diff
@@ -64,16 +64,14 @@ fi
 # You might need to install liblapack-dev package as well
 # sudo apt-get install -y liblapack-dev
 if [ ! -d "numpy" ]; then
```
Comment (ogrisel) [inline scripts/install-deeplearning-libraries.sh]: This should probably be changed to:

``` bash
 if [ ! -f "~/.numpy-site.cfg" ]; then
```

and then the `pip install numpy` command should be moved to after the `fi` to only protect the symlink creation and let the pip upgrade numpy to the latest version if possible with:

``` bash
pip install --upgrade numpy
```

**Comment by ogrisel** [comment]
The remarks hold for scipy. Also please let me know if you can run this successfully:
- on an instance where you delete the venv folder and the numpy folder
- on an instance where numpy has already been installed (nothing should happen but the script should not crash)

File: scripts/install-deeplearning-libraries.sh (hunk 1 above)
Comment (cancan101) [inline scripts/install-deeplearning-libraries.sh]: addressed.

File: scripts/install-deeplearning-libraries.sh [hunk 2]
```diff
@@ -62,18 +62,16 @@ fi
 # Build numpy from source against OpenBLAS
 # You might need to install liblapack-dev package as well
 # sudo apt-get install -y liblapack-dev
-if [ ! -d "numpy" ]; then
-    git clone -q --branch=v1.9.1 git://github.com/numpy/numpy.git
-    ln -s dl-machine/numpy-site.cfg numpy/site.cfg
-    (cd numpy && python setup.py install)
+if [ ! -d "~/.numpy-site.cfg" ]; then
```
Comment (ogrisel) [inline scripts/install-deeplearning-libraries.sh]: `-d` is for testing the presence of directories. Here `.numpy-site.cfg` is a symlink to a file. I think we need `-x` here. Could you please check that this works on a new instance?

Comment (ogrisel) [inline scripts/install-deeplearning-libraries.sh]: I meant `-f` instead of `-x`.

Comment (ogrisel) [inline scripts/install-deeplearning-libraries.sh]: Or maybe even just `-e`.

**Comment by ogrisel** [comment]
LGTM, have you checked that the scripts run correctly and that both numpy and scipy are built against openblas?

**Comment by ogrisel** [comment]
Merging to test it and re-create new snapshots.

**Pull request merged**